README.md

```diff
@@ -120,7 +120,10 @@ csrutil enable
     sudo cp ~/Downloads/SidecarCore.bak /System/Library/PrivateFrameworks/SidecarCore.framework/Versions/A/SidecarCore
     ```
     
-    Then re-enable System Integrity Protection (step 9). Your system should be in the same state as before you applied the patch!
+    * macOS 10.15.4+: Run `sudo nvram -d boot-args`
+    * Earlier macOS 10.15 versions: re-enable System Integrity Protection (step 9).
```

Comment:
we were missing how to revert the boot-args thing and it sounded like SIP was a thing regardless of the macOS version; for now I've just put it the different instructions in separate bullet points